lib/workers/miq_worker_types.rb

```diff
@@ -53,6 +53,7 @@
   "ManageIQ::Providers::Redhat::NetworkManager::EventCatcher"                   => %i(manageiq_default),
   "ManageIQ::Providers::Redhat::NetworkManager::MetricsCollectorWorker"         => %i(manageiq_default),
   "ManageIQ::Providers::Redhat::NetworkManager::RefreshWorker"                  => %i(manageiq_default),
+  "ManageIQ::Providers::Kubevirt::InfraManager::RefreshWorker"                  => %i(manageiq_default),
```

Comment:
It appears that up to line 63 there was an attempt to maintain this map ordered by key. Could you move this after   "ManageIQ::Providers::Kubernetes::MonitoringManager::EventCatcher" ?